standard/namespaces.md

```diff
@@ -446,10 +449,130 @@ Because names may be ambiguous when more than one imported namespace introduces
 > ```
 > *end example*
 
+Furthermore, when more than one namespace or type imported by *using_namespace_directive*s or *using_static_directive*s in the same compilation unit or namespace body contain types or members by the same name, references to that name as a *simple_name* are considered ambiguous.
+
+> *Example*:
+> ```csharp
+> namespace N1
+> {
+>     class A {}
+> }
+> class C
+> {
+>     public static int A;
+> }
+> namespace N2
+> {
+>     using N1;
+>     using static C;
+>     class B
+>     {
+>         void M() 
```

Comment:
We normally use brace-at-end-of-line for method declarations. (I know, I know...)